js_modules/dagster-ui/packages/ui-core/src/launchpad/LaunchpadSession.tsx

```diff
@@ -429,15 +440,19 @@ const LaunchpadSession: React.FC<LaunchpadSessionProps> = (props) => {
 
   const onSelectPreset = async (preset: Preset) => {
```

Comment:
@smackesey is this even a codepath that you can hit in dagit anymore

Reply:
Hmm I'm not sure about this one -- I think the question is whether `pipeline.presets[]` is ever populated in the GraphQL layer anymore. if it's 100% deprecated I'd love to clean this code up more broadly.

Reply:
according to tsserver `onSelectPreset` is not referenced anywhere